lib/elixir/lib/module.ex

```diff
@@ -782,16 +803,17 @@ defmodule Module do
   ## Examples
 
       defmodule Example do
-        def version, do: 1
+        defmacro version, do: 1
         Module.definitions_in __MODULE__, :def  #=> [{:version, 0}]
-        Module.definitions_in __MODULE__, :defp #=> []
+        Module.definitions_in __MODULE__, :defmacro #=> []
```

Comment:
I was probably figuring out the spec and played with the example and accidentally saved it.